lib/Issuing/Dispute.php

```diff
@@ -16,7 +16,7 @@
  *
  * @property string $id Unique identifier for the object.
  * @property string $object String representing the object's type. Objects of the same type share the same value.
- * @property \Stripe\BalanceTransaction[] $balance_transactions List of balance transactions associated with this dispute.
+ * @property null|\Stripe\BalanceTransaction[] $balance_transactions List of balance transactions associated with this dispute.
```

Comment:
Why doesn't that one have `bank_name` for ACSS like the other one @richardm-stripe

Reply:
ah nervemind it's only top-level properties that's why